src/components/HomeAgeVerification.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
Will the answer be stored in local storage? Will the user be able to navigate to the other pages if they refuse to answer or click no?